extensions/plugins/system/redcore/redcore.php

```diff
@@ -90,6 +90,11 @@ public function onAfterInitialise()
 						$data = RApi::getPostedData();
 						$dataGet = $input->get->getArray();
 
+						if (!empty($data['language']) && empty($dataGet['language']))
```

Comment:
this is not right...
there is a option to turn off translation, and with this you are disabling that option and making translation mandatory...
System already takes care of detecting should it translate or not...